packages/rrweb-snapshot/test/css.test.ts

```diff
@@ -119,6 +119,64 @@ describe('css parser', () => {
     expect(out3).toEqual('[data-aa\\:other] { color: red; }');
   });
 
+  it.each([
+    ['.foo,.bar {}', ['.foo', '.bar']],
+    ['.bar:has(:disabled) {}', ['.bar:has(:disabled)']],
+    [
+      '.bar:has(input:is(:disabled),button:has(:disabled)) {}',
+      ['.bar:has(input:is(:disabled),button:has(:disabled))'],
+    ],
```

Comment:
Lets also add a slightly simper test, one set of params with a comma in the middle:
```suggestion
    [
      '.bar:has(input, button) {}',
      ['.bar:has(input,button)'],
    ],
    [
      '.bar:has(input:is(:disabled),button:has(:disabled)) {}',
      ['.bar:has(input:is(:disabled),button:has(:disabled))'],
    ],
```